src/App.js

```diff
@@ -1,46 +1,21 @@
 
 import "./App.css";
-import { Instructions, Modal } from "./components";
+import { FileUpload, Instructions, Modal } from "./components";
 import { useState } from "react";
 import Papa from "papaparse";
 import generateMShotsUrl from "./lib/mShots";
-
-const Header = (props) => {
-  return (
-    <header className="app-header">
-      <h1>{props.title}</h1>
-      <p>{props.message}</p>
-    </header>
-  );
-}
-
-const Data = (props) => {
-  const data = props.data;
-  const getTotalPassed = () => {
-    return data.filter((item) => item.status === "PASSED").length;
-  }
-
-  const getTotalFailed = () => {
-    return data.filter((item) => item.status === "FAILED").length;
-  }
-  return(
-    <div>
-      <p>Number of items: {props.data.length}</p>
-      <p>Total passed: {getTotalPassed()}</p>
-      <p>Total failed: {getTotalFailed()}</p>	
-    </div>
-   )
-}
+import { REQUIRED_HEADERS } from "./constants/constants";
 
 const dataIsSet = (data) => data && data.length > 0
 
 const Home = (props) => {
 	const [usemShots, setUsemShots] = useState(false);
+  const [fileName, setFileName] = useState("No file chosen");
 	const handleChange = (event) => {
```

Comment:
**Suggestion (non-blocker)**: Now that this is being passed as a prop, I would recommend wrapping it with `useCallback`!

```typescript
const handleChange = useCallback((event) => { /*...*/}, [props.setData])
```

Reply:
> Now that this is being passed as a prop, I would recommend wrapping it with useCallback!

Now I understand it better. I think I am getting a hang of React 😀 thank you! Updated in 
[4bd30f9](https://github.com/Automattic/compare-snapshot-tool/pull/5/commits/4bd30f9cdb284a6bc4153739b557a2f3868ef67e).